lua/starfall/libs_cl/bass.lua

```diff
@@ -86,17 +152,49 @@ function bass_library.loadFile(path, flags, callback)
 				snd:Stop()
 				plyCount:free(instance.player, 1)
 			else
-				sounds[snd] = true
+				table.insert(soundList, snd)
+				soundDatas[snd] = { -- IGModAudioChannel is userdata, so we can't attach extra key-value pairs or functions to it directly.
+					flags = flags,
+					targetVolume = 1,
+					fadeMult = 1,
+					fadeMax = 5000,
+					outOfEarshot = false,
+					simpleFadeEnabled = false
+				}
+
+				-- Default with simple fade settings
+				if not is2D then
+					instance:runFunction(bass_methods.setFade, wrap(snd), 200, 5000, true)
```

Comment:
This can just be `addSoundToSimpleFade(snd)`

and you can put `snd:Set3DFadeDistance(200, 200)` inside addSoundToSimpleFade(snd)

Reply:
`bass_methods:setFade()` has some additional logic to it now, handling the dynamic clamp and setting the 3D fade. `add/remove SimpleFade` Are purely for tracking/untracking sounds, they shouldn't be burdened with setting other properties as well.

Setting the fade distance manually and calling `addSoundToSimpleFade(snd)` here during sound creation would also be unclean, as it would be a hard-scripted snippet of `bass_methods:setFade()`, which could cause confusion for future maintenance.

Reply:
I don't see any additional logic in setFade (yes I looked in setFade before making this suggestion). Also I don't see any reason not to put Set3DFadeDistance in addSoundToSimpleFade